docs/src/transports/aws-cloudwatch-logs.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Amazon CloudWatch Logs for LogLayer
+description: Send logs to Amazon CloudWatch platform with the LogLayer logging library
+---
+
+# Amazon CloudWatch Logs Transport <Badge type="tip" text="Server" /> <Badge type="warning" text="Browser" />
+
+[![NPM Version](https://img.shields.io/npm/v/%40loglayer%2Ftransport-aws-cloudwatch-logs)](https://www.npmjs.com/package/@loglayer/transport-aws-cloudwatch-logs)
+
+[Transport Source](https://github.com/loglayer/loglayer/blob/master/packages/transports/aws-cloudwatch-logs)
+
+The Amazon CloudWatch Logs transport allows you to send logs to [Amazon CloudWatch](https://aws.amazon.com/cloudwatch/), a service to monitor and manage resources in AWS.
+It uses the [AWS SDK for JavaScript CloudWatchLogs](https://www.npmjs.com/package/@aws-sdk/client-cloudwatch-logs).
+
+## Installation
+
+::: code-group
+
+```sh [npm]
+npm install loglayer @loglayer/transport-aws-cloudwatch-logs @aws-sdk/client-cloudwatch-logs
+```
+
+```sh [pnpm]
+pnpm add loglayer @loglayer/transport-aws-cloudwatch-logs @aws-sdk/client-cloudwatch-logs
+```
+
+```sh [yarn]
+yarn add loglayer @loglayer/transport-aws-cloudwatch-logs @aws-sdk/client-cloudwatch-logs
+```
+
+:::
+
+## Usage
+
+```typescript
+import { LogLayer } from 'loglayer';
+import { CloudWatchLogsTransport } from "@loglayer/transport-aws-cloudwatch-logs";
+
+// Create LogLayer instance with CloudWatch Logs transport
+const log = new LogLayer({
+  transport: new CloudWatchLogsTransport({
+    groupName: "/loglayer/group",
+    streamName: "loglayer-stream-name",
+  }),
+});
+
+// Use LogLayer as normal
+log.withMetadata({ customField: 'value' }).info('Hello from Lambda!');
+```
+
+### Server-Side Usage
```

Comment:
since we've agreed to not bother with browser, we can consolidate this into the main usage